### PR TITLE
adds initial docs around funding and best practices

### PR DESCRIPTION
**Pull request by nellshamrell**

### CODE REVIEW

File: docs/best-practice-non-memory-safe-by-default-languages.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Best Practices - Non Memory-Safe By Default Languages
+
+This working group recommends switching to a memory-safe by default language whenever possible. However, when switching to a memory-safe by default language is not possible, there are developer practices that will improve memory safety, even if it is not the default.
```
Comment (GabrielDosReis) [inline docs/best-practice-non-memory-safe-by-default-languages.md:3]: suggestion: "whenever possible or practical"

**Review by GabrielDosReis — APPROVED** [review]

File: docs/best-practice-non-memory-safe-by-default-languages.md (hunk 1 above)
Comment (nellshamrell) [reply docs/best-practice-non-memory-safe-by-default-languages.md]: Added!

File: docs/best-practice-non-memory-safe-by-default-languages.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Best Practices - Non Memory-Safe By Default Languages
+
+This working group recommends switching to a memory-safe by default language whenever possible or practical. However, when switching to a memory-safe by default language is not possible or practical, there are developer practices that will improve memory safety, even if it is not the default.
+
```
Comment (balteravishay) [inline docs/best-practice-non-memory-safe-by-default-languages.md:4]: should this also point to the BEST c/c++ hardening guide: https://github.com/ossf/wg-best-practices-os-developers/tree/main/docs/Compiler-Hardening-Guides

**Review by balteravishay — APPROVED** [review]
NIT

**Comment by edelsohn** [comment]
Suggest "This working group recommends **using** a memory-safe by default language whenever possible or practical.

File: docs/best-practice-non-memory-safe-by-default-languages.md (hunk 2 above)
Comment (nellshamrell) [reply docs/best-practice-non-memory-safe-by-default-languages.md:4]: Done!

**Comment by nellshamrell** [comment]
Suggested edits have been made :)

**Review by GabrielDosReis — APPROVED** [review]

**Pull request merged**